lbuild/facade.py

```diff
@@ -194,19 +239,32 @@ def substitutions(self, substitutions):
         self._env.substitutions = substitutions
 
     def copy(self, src, dest=None, ignore=None, metadata=None):
-        self._env.copy(src, dest, ignore, metadata)
+        if metadata:
+            deprecated("1.8.0", "EnvironmentBuild.copy(..., metadata=metadata)",
+                                "EnvironmentBuild.collect(..., scope=scope)")
```

Comment:
I guess the method should still be called `copy` and not `collect`?

Reply:
Nah, I guess this would be the correct way, I'll add some nice formatting for multiple lines.
```python
deprecated("1.8.0", "EnvironmentBuild.copy(..., metadata=metadata)",
                    "scope = env.copy(...)\nfor key, values in metadata.items()\nenv.collect(key, *values, scope=scope)")
```

Reply:
Ok, it's not super pretty, but this is how it works now:
```
[WARNING] lbuild.facade: 'EnvironmentBuild.copy(..., metadata=metadata)' is deprecated since v1.8.0, use
    scope = env.copy(...)
    for key, values in metadata.items():
        env.collect(key, *values, scope=scope)
instead!!
```

Reply:
done, with `s/scope/operations/g`.